### ISSUE TITLE
Panzer:  Incorrect #include of uneeded ml_rbm.h 

### ISSUE THREAD

**Issue by glhenni** [trilinos/Trilinos#11408]
## Bug Report
@rppawlo 

I'm trying to pare down the ENABLE and dependencies for charon and in the process I am getting  an error about `ml_rbm.h` not being found when compiling `adapters-stk/src/Panzer_STK_SetupLOWSFactory.cpp`. First, that **#include** is contained within what I believe is an incorrect ifdef, *PANZER_HAVE_EPETRA*. Seems like that should be something about having ML not EPETRA. 

Second, the code that uses the things defined in that include file is commented out starting at around line 225 of `Panzer_STK_SetupLOWSFactory.cpp` so at the very least that include should also be commented out.

**Comment by GrahamBenHarper** [comment]
Hi @glhenni, are you using the develop branch of Trilinos?

**Comment by glhenni** [comment]
> Hi @glhenni, are you using the develop branch of Trilinos?

I am using the develop branch

**Comment by GrahamBenHarper** [comment]
It's possible this is related to #11402 or #11405 and the recent change to make Epetra optional. Roger's already working on a fix, and we'll keep you updated when a pull request goes through.

**Comment by glhenni** [comment]
Oh, it definitely is related. In the process of fixing that for our project, charon, I was going to clean up our use of <whatevever>_ENABLE_<package> and instead the appropriate packages to LIB_REQUIRED_DEP_PACKAGES and I ran across this one, which I don't think has to do directly with the EPETRA work but was exposed by it.

**Comment by GrahamBenHarper** [comment]
Are you still seeing any issues with Charon since the PR merged?

**Comment by glhenni** [comment]
I just included "ML" as a dependency for charon to get around this problem, so the answer is, no, I am no longer seeing this problem, regardless of the PR merge. I suspect I would see it again if I removed the "ML" dependency though because it doesn't look like the source code causing the problem was changed, unless someone added an "ML" dependency.

I would still argue that `#include "ml_rbm.h"` should *not* be in the `PANZER_HAVE_EPETRA_STACK` ifdef block, and isn't needed at all right now because the related ML code is commented out starting at around line 226 of *Panzer_STK_SetupLOWSFactory.cpp*. At the very least commenting out the line `#include "ml_rbm.h"` would be consistent with the fact that the ML code is commented out.
